recipe/meta.yaml

```diff
@@ -546,13 +549,8 @@ outputs:
       run_constrained:
         - {{ pin_subpackage("libgomp", exact=True) }}
         - msys2-conda-epoch <0.0a0    # [win]
-    build:
-      number: {{ build_num }}
-      detect_binary_files_with_prefix: False
-      skip: True   # [target_platform != cross_target_platform]
-      missing_dso_whitelist:
-        - "*"
-      rpaths_patcher: patchelf
+        # avoid installing incompatible version of old name for this output
+        - libgcc-ng =={{ version }}
```

Comment:
```suggestion
        - libgcc-ng {{ version }}.* *_{{ build_num }}
```

Reply:
Otherwise we can have the old `libgcc-ng` with the same verison number interfere.

Reply:
Good point.